app/models/katello/host/subscription_facet.rb

```diff
@@ -41,20 +41,15 @@ def update_subscription_status
       end
 
       def self.new_host_from_rhsm_params(params, org, location)
-        facts = params[:facts]
-        fqdn = facts['network.hostname']
-
-        ::Host::Managed.new(:name => fqdn, :organization => org, :location => location,
-                          :managed => false, :interfaces => new_interfaces(facts))
+        fqdn = params[:facts]['network.hostname']
```

Comment:
Seems like we could just pass `params[:facts]` into this method and avoid this model level class from having to know about "params" and extracting `:facts` from it.